pennylane/pulse/rydberg_hamiltonian.py

```diff
@@ -0,0 +1,236 @@
+# Copyright 2018-2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module contains the classes/functions needed to simulate the evolution of ensembles of
+individual (trapped) rydberg atoms under the excitation of several laser fields."""
+import warnings
+from dataclasses import dataclass
+from typing import Callable, List, Union
+
+import numpy as np
+
+import pennylane as qml
+from pennylane.ops import SProd
+from pennylane.wires import Wires
+
+from .parametrized_hamiltonian import ParametrizedHamiltonian
+
+
+def rydberg_interaction(register: list, wires=None, interaction_coeff: float = 862690 * 2 * np.pi):
```

Comment:
I'm still unsure about having a specific value for the interaction coefficient.

Reply:
What would you consider doing instead?

Reply:
I find it quite useful to have good defaults, so a user that just wants to simulate a realistic system without having to dive into the specifics of the hardware manufacturer and device can conveniently use this Hamiltonian for their pulse program

Reply:
Was the outcome that we want something with a realistic order of magnitude, but not a specific value? I.e. should this default be 10e5? Then where/how do we provide information about the accurate value for f.x. the QuEra machine?

Reply:
My understanding was that there would be device specific methods `dev.rydberg_interaction()` with the appropriate values. A good question is where this is documented then 🤔 In the plugin? In `qml.devices`? Both sound sub-optimal